comps/cores/mega/utils.py

```diff
@@ -258,3 +261,73 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         if exc_type:
             self.context_to_manage.__exit__(exc_type, exc_val, exc_tb)
+
+
+def _handle_message(messages):
```

Comment:
what's it for? from naming convention, it has an underscore prefix and act as a internal func. but nobody uses it?

Reply:
many GenAIExamples need the function